<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatPageWidget</class>
 <widget class="QWidget" name="ChatPageWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="1,10,1">
     <item alignment="Qt::AlignHCenter">
      <widget class="QWidget" name="HeadBarWidget" native="true">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="nameLabel">
          <property name="text">
           <string>还未开始聊天</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="MessageListWidget" name="MessageList"/>
     </item>
     <item>
      <widget class="QWidget" name="InputBarWidget" native="true">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="leftMargin">
         <number>3</number>
        </property>
        <property name="topMargin">
         <number>3</number>
        </property>
        <property name="rightMargin">
         <number>3</number>
        </property>
        <property name="bottomMargin">
         <number>3</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="uploadButton">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="Easy.qrc">
              <normaloff>:/Easy/src/icon/upload_file.svg</normaloff>:/Easy/src/icon/upload_file.svg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>35</width>
              <height>35</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="textContent"/>
          </item>
          <item>
           <widget class="QPushButton" name="sendButton">
            <property name="text">
             <string>发送</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MessageListWidget</class>
   <extends>QListWidget</extends>
   <header>MessageListWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Easy.qrc"/>
 </resources>
 <connections/>
</ui>
